### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Janglinator** [herzbube/littlego#407] [General]
Just wanted to drop by and say thanks!

New Go player here, found Little Go through the App Store.  As an iOS dev, I was naturally curious when I saw it’s open source, so I dug around a bit!  It seems like the repo is your own personal echo chamber when looking ant issues, contributions, discussions,…. well frankly everything :)

It was fun seeing your project and how well you’ve maintained and organized the project through the years.  Even if it is still ObjC ;)

Wanted to say thanks for your hard work!

**Comment by herzbube** [comment]
Hey Jangles, apologies for the late reply, and thanks for the kind words, very much appreciated!

Although there were some very valuable contributions (the logo, bug reports), this has indeed been mostly a solo project. While this has some advantages (I'm the ruler of the universe ☺), it would be nice to collaborate with someone who has fresh energy to push some topics (e.g. an UX expert revamping the UI, making a tutorial, playing over the network, integrating one of the new shiny Go engines). Well, it is as it is, so people just have to be patient with me as I'm slowly wading through the backlog while also keeping the project alive.

> Even if it is still ObjC ;)

Hah, Objective-C is such a nice language, I'm sad to see it slowly fading. One of these days I will try out Swift, but this project will certainly not be my personal test balloon. That being said, I believe it's possible to mix Objective-C and Swift in one project, so if a Swift enthusiast wants to contribute here and can show how to achieve co-existence of the two languages, then I will not be the one who says no.

Cheers, Patrick
